opa-react-demo/docker-compose.yaml

```diff
@@ -8,6 +8,7 @@ services:
 
   eopa:
     image: ghcr.io/styrainc/enterprise-opa:latest
+    pull_policy: always
```

Comment:
Just a thing I've found today: it'll make `docker compose up` always check for the latest image first.